examples/cached_alias_config.py

```diff
@@ -33,6 +33,7 @@
     assert await cache.get("key") == "value"
     assert isinstance(cache, SimpleMemoryCache)
     assert isinstance(cache.serializer, DefaultSerializer)
+    await cache.close()
```

Comment:
The user can choose to close this cache whenever they want.  This is just an example usage, no?